test/upgrade/upgrade_test.sh

```diff
@@ -13,65 +13,100 @@
 # distributed under the License is distributed on an "AS IS" BASIS,
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
-# limitations under the License.
+ # limitations under the License.
 
 
+# This script test upgrading from Accumulo 1.9 to 2.0. This script is
+# not self verifying, its output must be inspected for correctness.
+
 if [[ $# != 1 ]] ; then
   BASENAME=$(basename "$0")
   echo "Usage: $BASENAME clean|dirty"
   exit -1
 fi
 
-#this script test upgrade.   This script is not self verifying, its output must be inspected for correctness.
-
-#set DIR  to point to configured accumulo dirs. 
-
-#TODO could support multinode configs, this script assumes single node config
-
-PREV=../../../accumulo-1.7.1
-CURR=../../
-DIR=/accumulo
+# Dir of local git repo containing Accumulo
+ACCUMULO_DIR=~/git/accumulo
+# Dir where Uno is located
+UNO_DIR=~/git/uno
+# HDFS dir were bulk import files are generated
 BULK=/tmp/upt
-INSTANCE=testUp
 
-pkill -f accumulo.start
-hadoop fs -rmr "$DIR"
-hadoop fs -rmr "$BULK"
-hadoop fs -mkdir -p "$BULK/fail"
+# This script assumes Uno is configured to build Accumulo from
+# $ACCUMULO_DIR when 'uno fetch accumulo' is run.
+
+cd $ACCUMULO_DIR
+git checkout 1.9
+git clean -xfd
+cd $UNO_DIR
+./bin/uno fetch accumulo
+./bin/uno setup accumulo
+(
```

Comment:
Why a subshell?

Reply:
The comment on the next line states why, maybe its not enough.  Running `eval "$(./bin/uno env)"` sets a bunch of env vars that are related to the old version of accumulo

Reply:
Ah, okay, I get it now, it's because you want to isolate the environment for the old version from the environment for the new version. I get it now. That wasn't clear to me the first time. Thanks.

Reply:
I updated the comment to explain this.